tools/AutomationTestFramework/ManageScripts/GenerateRunbook.ps1

```diff
@@ -0,0 +1,122 @@
+. "$PSScriptRoot\ListTestFunctions.ps1"
+
+<#
+.SYNOPSIS
+Generate a powershell file with basic script to run tests.
+.DESCRIPTION
+Collects list of test function names that marked as AzureAutomation in dot Description section and adds them to powershell script.
+.PARAMETER srcPath
+Path to the solution src folder.
+.PARAMETER bookName
+Name of the book on Automation Account.
+.PARAMETER connectionName
+Name of connection to use in the runbook.
+.PARAMETER outputPath
+Folder path to put generated runbook to.
+If not specified, the value will be '..\RunBooks'
+.EXAMPLE
+GenerateRunbook "e:\git\azure-powershell\src\ResourceManager\Storage\Commands.Management.Storage.Test\ScenarioTests\" "ShchBook" "ShchConn".
+#>
+function GenerateRunbook (
+     [string] $srcPath
+    ,[string] $bookName
+    ,[string] $connectionName
+    ,[string] $outputPath) {
+
+    if ([string]::IsNullOrEmpty($outputPath)) {
+        $outputPath = Join-Path "$PSScriptRoot\..\" "RunBooks"
+    }
+
+    $bookPath = Join-Path $outputPath "$bookName.ps1"
+    $bookPath
+    $null = New-Item $bookPath -type file -Force
+
+    # These params to save result streams in Azure Storage Account Container
+    "Param("                                | Add-Content $bookPath
```

Comment:
It would be nice if the parameterized contents of the script were contained in an external file (template) in this directory, and this code simply replaced the function list and the connectionName from the script.  This would make the runbook script a bit easier to edit.  We follow this pattern for other files, liek the psm1 example template here: https://github.com/Azure/azure-powershell/blob/preview/tools/AzureRM.Example.psm1

where we use the pattern:
```%TemplateProeprty%``` to indicate a value we replace in a script.

Reply:
done.